marklogic-data-hub-central/ui/src/api/__mocks__/mocks.data.ts

```diff
@@ -361,6 +361,20 @@ const clearUserDataAPI = (axiosMock) => {
   });
 };
 
+const getAllExcludeValuesList = (axiosMock) => {
+  return  axiosMock.get["mockImplementation"]((url) => {
+    switch (url) {
+    case "/api/steps/matching/exclusionList":
+      return Promise.resolve({
+        "data": {"name": "Preset List 1", "values": ["uno", "dos", "uno", "dos", "uno", "dos", "uno", "dos"]},
```

Comment:
i get what you meant to do here but i think we should stick to the same language through the app :)